### PR TITLE
PIL-1086: ensure native tokens for all chains

### PR DESCRIPTION
**Pull request by mdjastrzebski**
Scope:
- [x] Append native tokens if not present in supported tokens list
- [x] Central definition for nativeAssetPerChain
- [x] Removed nativeAssetSymbolPerChain & nativeAssetDecimalsPerChain

Note: there is some overlap with supported assets per chain, left comments in the code to help address that.

### CODE REVIEW

File: src/services/etherspot.js [hunk 1]
```diff
@@ -583,22 +588,13 @@ export class EtherspotService {
...
-          decimals: 18,
-          iconUrl,
-          iconMonoUrl: iconUrl,
-        });
-      }
+      // TODO: replace by single invocation per chain:
+      supportedAssets = appendNativeAssetIfNeeded(CHAIN.ETHEREUM, supportedAssets);
+      supportedAssets = appendNativeAssetIfNeeded(CHAIN.POLYGON, supportedAssets);
+      supportedAssets = appendNativeAssetIfNeeded(CHAIN.XDAI, supportedAssets);
+      supportedAssets = appendNativeAssetIfNeeded(CHAIN.BINANCE, supportedAssets);
```
Comment (mdjastrzebski) [inline src/services/etherspot.js:597]: This part needs to be addresses in this PR or supported assets per chains (whichever merges last).

File: src/utils/etherspot.js [hunk 2]
```diff
@@ -204,31 +204,12 @@ export const buildExchangeOffer = (
...
-    address: EthersConstants.AddressZero,
-    name: 'Ethereum', // eslint-disable-line i18next/no-literal-string
-    symbol: ETH,
-    decimals: 18,
-    iconUrl,
-    iconMonoUrl: iconUrl,
-  };
-
-  return [eth, ...assets];
+  // TODO: Switch to zero address check when supported assets per chain are implemented.
```
Comment (mdjastrzebski) [inline src/utils/etherspot.js:210]: This part should to be addresses in this PR or supported assets per chains (whichever merges last).

**Review by IAmKio — APPROVED** [review]
LG - made a mental note of the TODO's 👍

**Review by poocart — APPROVED** [review]
LGTM

**Pull request merged**